server/router/index.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express');
-const { logout } = require('../controllers');
+const { logout, classActivities } = require('../controllers');
 const { auth } = require('../controllers/middlewares');
 
 const router = express.Router();
 
+
 router.use(auth);
+router.get('/subject/:subjectId/activities/:classId', classActivities);
```

Comment:
that's a good design, just use plural for `subjects` - the idea is you have  collection of resources (plural) / :id of concrete resource